src/core/src/enclave_sim.rs

```diff
@@ -141,8 +141,7 @@ pub fn initialize_wait_certificate(
     if !in_serialized_prev_block_wait_certificate.is_empty() {
         let deserialized_prev_block_wait_certificate =
                serde_json::from_str(&in_serialized_prev_block_wait_certificate);
-        let mut prev_wait_certificate_obj : WaitCertificate =
-               WaitCertificate::default();
+        let prev_wait_certificate_obj : WaitCertificate;
```

Comment:
Please revert to the original line. This one will error out, as we're modifying the var in next few lines.